crates/runtime/src/runtime.rs

```diff
@@ -26,8 +33,12 @@ impl RuntimeBuilder {
         let mut engine_config = wasmtime::Config::default();
         engine_config.async_support(true);
         engine_config.wasm_component_model(true);
+        engine_config.memory_init_cow(false);
+
         Self {
             engine_config,
+            max_components: 10000,
+            max_component_size: 15 * MB,
```

Comment:
I think we might want to make this larger, the out-of-the-box hello world component for Python is 35mb
```bash
➜ ll build
Permissions Size User   Date Modified Name
.rw-r--r--     0 brooks  4 Mar 10:14  .gitkeep
.rw-r--r--   35M brooks  4 Mar 10:30  http_hello_world.wasm
.rw-r--r--   35M brooks  4 Mar 10:30  http_hello_world_s.wasm
```

I'm not sure what would balloon this up to a larger number, and since our TS/JS component is only 9mb Python is the only one that exceeds this. Thoughts on at least upping this to 40MB, or maybe 64MB for a computer number?